includes/wikia/Wikia.php

```diff
@@ -1964,11 +1964,15 @@ public static function onWikiaSkinTopScripts( &$vars, &$scripts, Skin $skin ) {
 	/**
 	 * Hook for storing historical log of email changes
 	 * Depends on the central user_email_log table defined in the EditAccount extension
+	 *
+	 * @param User $user
+	 * @param $new_email
+	 * @param $old_email
 	 * @return bool
 	 */
 	public static function logEmailChanges($user, $new_email, $old_email) {
 		global $wgExternalSharedDB, $wgUser, $wgRequest;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)

[project-id]: wikia:app
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)

[project-id]: wikia:app